[2024-12-22T14:24:11.036+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: TEST_1.run_bash_command manual__2024-12-22T14:24:09.769295+00:00 [queued]>
[2024-12-22T14:24:11.044+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: TEST_1.run_bash_command manual__2024-12-22T14:24:09.769295+00:00 [queued]>
[2024-12-22T14:24:11.045+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2024-12-22T14:24:11.056+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): run_bash_command> on 2024-12-22 14:24:09.769295+00:00
[2024-12-22T14:24:11.062+0000] {standard_task_runner.py:57} INFO - Started process 52 to run task
[2024-12-22T14:24:11.066+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'TEST_1', 'run_bash_command', 'manual__2024-12-22T14:24:09.769295+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/tets.py', '--cfg-path', '/tmp/tmpkc4wvcw2']
[2024-12-22T14:24:11.068+0000] {standard_task_runner.py:85} INFO - Job 3: Subtask run_bash_command
[2024-12-22T14:24:11.084+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-12-22T14:24:11.124+0000] {task_command.py:415} INFO - Running <TaskInstance: TEST_1.run_bash_command manual__2024-12-22T14:24:09.769295+00:00 [running]> on host d191bdd92ba4
[2024-12-22T14:24:11.216+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='TEST_1' AIRFLOW_CTX_TASK_ID='run_bash_command' AIRFLOW_CTX_EXECUTION_DATE='2024-12-22T14:24:09.769295+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-22T14:24:09.769295+00:00'
[2024-12-22T14:24:11.219+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-12-22T14:24:11.219+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python3 /opt/***/DE_project/hello.py']
[2024-12-22T14:24:11.231+0000] {subprocess.py:86} INFO - Output:
[2024-12-22T14:24:11.325+0000] {subprocess.py:93} INFO - heloooooooooooooo
[2024-12-22T14:24:11.330+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-12-22T14:24:11.362+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=TEST_1, task_id=run_bash_command, execution_date=20241222T142409, start_date=20241222T142411, end_date=20241222T142411
[2024-12-22T14:24:11.409+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-12-22T14:24:11.435+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
